desktop/plugins/network/MockResponseDetails.tsx

```diff
@@ -322,31 +349,11 @@ export function MockResponseDetails({id, route, isDuplicated}: Props) {
               stickyBottom={true}
               autoHeight={true}
               floating={false}
-              // height={300}
```

Comment:
Does this fix https://github.com/facebook/flipper/issues/1460? Otherwise Wrapping a `Layout.ScrollContainer` could fix that I'd expect

Reply:
Yes, this does resolve https://github.com/facebook/flipper/issues/1460.